sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/JsonToRow.java

```diff
@@ -334,27 +335,27 @@ private ObjectMapper objectMapper() {
   @AutoValue
   public abstract static class ParseResult implements POutput {
 
-    public abstract JsonToRowWithErrFn getJsonToRowWithErrFn();
+    abstract JsonToRowWithErrFn getJsonToRowWithErrFn();
 
-    public abstract PCollection<Row> getParsedLine();
+    abstract PCollection<Row> getParsedLine();
 
-    public abstract PCollection<Row> getFailedParse();
+    abstract PCollection<Row> getFailedParse();
```

Comment:
Aren't these two the methods that users will be trying to access?
(others LGTM)

Reply:
There are two public methods:
https://github.com/apache/beam/pull/12175/files#diff-eef3983ed081455798dfacfa9251ff38L382

https://github.com/apache/beam/pull/12175/files#diff-eef3983ed081455798dfacfa9251ff38L393

That folks can use